packages/core/src/RenderingEngine/helpers/setDefaultVolumeVOI.ts

```diff
@@ -88,26 +88,30 @@ function getVOIFromMetadata(imageVolume: IImageVolume): VOIRange {
     const voiLutModule = metaData.get('voiLutModule', imageId);
     if (voiLutModule?.windowWidth && voiLutModule.windowCenter) {
       const { windowWidth, windowCenter } = voiLutModule;
-      voi = {
-        windowWidth: Array.isArray(windowWidth) ? windowWidth[0] : windowWidth,
-        windowCenter: Array.isArray(windowCenter)
-          ? windowCenter[0]
-          : windowCenter,
-      };
+      const width = Array.isArray(windowWidth) ? windowWidth[0] : windowWidth;
+      const center = Array.isArray(windowCenter)
+        ? windowCenter[0]
+        : windowCenter;
+
+      // Skip if both windowWidth and windowCenter are 0
```

Comment:
Don't you only care if the width is 0?  The center shouldn't matter if the width is zero - I mean it is defined, everything below center is black, and everything above or equal is white.  But that actually works at 0, 0 for width/center for positive/negative values.